proof/invariant-abstract/Detype_AI.thy

```diff
@@ -1042,7 +1042,7 @@ lemma (in Detype_AI) cte_map_not_null_outside: (*FIXME: arch-split*)
   apply (drule(1) valid_globals_irq_node, fastforce simp: cap_range_def)
   done
 
-
+\<comment> \<open>FIXME: move\<close>
```

Comment:
this and the next FIXME move are added as part of "aarch64 ainvs: update for spec changes" which is a bit odd.
also, nearly all our FIXME comments are in `(* FIXME ... *)` form, which makes grepping easier. Any reason why these ones are using the `― ‹FIXME: move›` version?

Reply:
Only because I have a keybind for commenting things out with `― ‹...›` and because I didn't know that people used the comment format for grepping for FIXMEs.